### DISCUSSION TITLE
Showing Backlinks for an Article

### DISCUSSION THREAD

**Discussion by jeremyf** [forem/forem#16159] [Feature Requests]
For building up a corpus of knowledge, it could be useful to show the backlinks within a Forem to a given Article.  That is to say, when Person A writes a great article.  And I write an Article in the same Forem that links to Person A's article, then it would be useful to see a backlink from Person A's article to my article.

**Comment by Harvey8426** [comment]
As for this process we are doing the manual internal linking of [small business email marketing](https://eliteemailsoftware.com/email-marketing-for-small-businesses/) forum. Which is helpful if someone is interested in reading the related article. Although according to my experience a large number of tools available in the market which are doing the same but in the forums you could also do it manually. The auto system is difficult in the forums.
